app/controllers/reviews_controller.rb

```diff
@@ -41,6 +38,22 @@ def create
     if @review.save
       flash[:notice] = "Review added"
       redirect_to bar_path(@bar)
+      mg_client = Mailgun::Client.new ENV["MAILGUN_API_KEY"]
```

Comment:
Can you abstract this into a private method? The private method can be in this controller